src/cordova/apple/OutlineAppleLib/Sources/OutlineTunnelSources/OutlineVpn.swift

```diff
@@ -12,19 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import CocoaLumberjack
 import CocoaLumberjackSwift
 import NetworkExtension
-import OutlineTunnel
 
 // Manages the system's VPN tunnel through the VpnExtension process.
 @objcMembers
-class OutlineVpn: NSObject {
-  static let shared = OutlineVpn()
+public class OutlineVpn: NSObject {
```

Comment:
Are there any reasons that we are changing the access level in this file from the [default `internal`](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/accesscontrol/#Default-Access-Levels) to `public`?